### PR TITLE
GWTP-119 : Write the Resources section

### PR DESCRIPTION
**Pull request by meriouma**
This is a kind of first draft. Thoughts?

gwtp-119

### CODE REVIEW

File: site/src/main/markdown/gwtp/resources/index.md [hunk 1]
```diff
@@ -1,418 +1,33 @@
...
+- GWTP Basic Sample \[[Live Demo](http://gwtp-sample-basic.appspot.com/)\] \[[Sources](https://github.com/ArcBees/GWTP-Samples/tree/master/gwtp-samples/gwtp-sample-basic)\]
+- GWTP & Spring Basic Sample \[[Live Demo](http://gwtp-sample-basic-spring.appspot.com/)\] \[[Sources](https://github.com/ArcBees/GWTP-Samples/tree/master/gwtp-samples/gwtp-sample-basic-spring)\]
+- GWTP's Nested Presenters \[[Live Demo](http://gwtp-sample-nested.appspot.com/)\] \[[Sources](https://github.com/ArcBees/GWTP-Samples/tree/master/gwtp-samples/gwtp-sample-nested)\] 
+- GWTP's Form factors \[[Live Demo](http://gwtp-sample-mobile.appspot.com/)\] \[[Sources](https://github.com/ArcBees/GWTP-Samples/tree/master/gwtp-samples/gwtp-sample-mobile)\]
+- GWTP's Tabbed Presenters \[[Live Demo](http://gwtp-sample-tab.appspot.com/)\] \[[Sources](https://github.com/ArcBees/GWTP-Samples/tree/master/gwtp-samples/gwtp-sample-tab)\]
+- GWTP's Crawler Service \[[Sources](https://github.com/ArcBees/GWTP-Samples/tree/master/gwtp-samples/gwtp-sample-crawler-service)\]
```
Comment (christiangoudreau) [inline site/src/main/markdown/gwtp/resources/index.md]: All those live demos works, right?

**Comment by christiangoudreau** [comment]
LGTM, simple is better :D

File: site/src/main/markdown/gwtp/resources/index.md (hunk 1 above)
Comment (meriouma) [inline site/src/main/markdown/gwtp/resources/index.md]: Yep

File: site/src/main/markdown/gwtp/resources/index.md [hunk 2]
```diff
@@ -1,418 +1,33 @@
-# Maven Configuration
+# ReSourcess
```
Comment (Chris-V) [inline site/src/main/markdown/gwtp/resources/index.md]: Couple typos here

**Comment by Chris-V** [comment]
:+1

**Comment by christiangoudreau** [comment]
I just realized that the animated gif animate in my email. I almost came.

**Pull request merged**